docroot/modules/custom/va_gov_post_api/src/Form/VaGovFacilityForceQueueForm.php

```diff
@@ -95,7 +110,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['facility_type'] = [
       '#type' => 'radios',
       '#title' => $this->t('Content type'),
-      '#description' => t('Select a content type to queue.'),
+      '#description' => $this->t('Select a content type to queue.'),
```

Comment:
I don't see this class using the string translation trait.  (maybe some higher extended class is bringing it in?)  
If this was not tested, it would be a breaking change.   Also this form is only for us admins, so it is a bit silly to translate this.
```suggestion
      '#description' => 'Select a content type to queue.',
```

Reply:
Removing `$this->t()` triggers a warning and returns non-zero status from phpcs during pre-commit.